Applications/MainApplication/Gui/MaterialEditor.cpp

```diff
@@ -95,57 +97,59 @@ void MaterialEditor::newKsColor( const QColor& color ) {
 
     if ( m_renderObject && m_usable )
     {
-        Core::Utils::Color c( color.redF(), color.greenF(), color.blueF(), 1.0 );
-        m_material->m_ks = c;
+        m_blinnphongmaterial->m_ks = Core::Utils::Color (
+              color.redF(), color.greenF(), color.blueF(), 1. );
```

Comment:
`1_ra` ?

Reply:
No, `blueF` and others are double, so we need a double here to have all the parameters with the same type (see `ColorBase` constructor)